src/Cli/dotnet/commands/dotnet-workload/clean/LocalizableStrings.resx

```diff
@@ -0,0 +1,137 @@
+    <xsd:element name="root" msdata:IsDataSet="true">
+      <xsd:complexType>
+        <xsd:choice maxOccurs="unbounded">
+          <xsd:element name="metadata">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" />
+              </xsd:sequence>
+              <xsd:attribute name="name" use="required" type="xsd:string" />
+              <xsd:attribute name="type" type="xsd:string" />
+              <xsd:attribute name="mimetype" type="xsd:string" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="assembly">
+            <xsd:complexType>
+              <xsd:attribute name="alias" type="xsd:string" />
+              <xsd:attribute name="name" type="xsd:string" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="data">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="CleanAllOptionDescription" xml:space="preserve">
+    <value>Clean up and uninstall all CLI workloads for every SDK feature band (version) that is less than or equal to the current running SDK.</value>
+  </data>
+  <data name="CommandDescription" xml:space="preserve">
+    <value>Remove orphaned workload components that may be on the machine.</value>
```

Comment:
Also instead of "orphaned" can we use a more precise term? What exactly does that mean here? A pack with no matching installer? Or the reverse? Or something else entirely?

Reply:
Ya. The word "orphaned" is not great.

> Also, should we say "workload packs" instead of "workload components"?

I don't think so. `clean` also cleans pack installation records. `clean --all` also cleans all of that + workload installation records. But I doubt most customers know what those are. Hence `components`.

> A pack with no matching installer? 

No matching installer or no matching workload installation record. Or if clean all, every pack.

Reply:
I should be more precise with my language. It will only clean the packs of the same SDK type (file based or MSI based if the SDK is in this mode.) And it will tell for VS workloads how to remove them in VS as it can't remove those. The current code also only removes for feature bands at or below the current band, but I'm about to change that limitation.

Reply:
Per @baronfel,
`Removes workload components from this SDK feature band that may have been left behind from previous updates and uninstallations.`

`--all` --> `Causes clean to remove all workload components from all SDK versions.`